tasks/gotest.py

```diff
@@ -943,3 +945,46 @@ def lint_go(
     only_modified_packages=False,
 ):
     raise Exit("This task is deprecated, please use `inv linter.go`", 1)
+
+
+@task
+def check_otel_dependencies(ctx):
+    with ctx.cd("test/otel"):
+        # Rename fixtures
+        ctx.run("mv dependencies.go.fake dependencies.go")
+        ctx.run("mv go.mod.fake go.mod")
+        ctx.run("mv go.sum.fake go.sum")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Can you add the followings to `.gitignore`

```
test/otel/*.go
test/otel/go.mod
test/otel/go.sum
```

 or add a `.gitignore` to `test/otel` with

```
*.go
go.mod
go.sum
```

This way if you run the command locally, you ignore the moved files and you avoid accidentally committing them